source

```diff
@@ -105687,15 +105703,21 @@ document.body.appendChild(text);
    <dt>A start tag whose tag name is one of: "head", "noscript"</dt>
    <dt>Any other end tag</dt>
    <dd>
-    <p><span>Parse error</span>. Ignore the token.</p>
+    <p>This is an <span
+    data-x="parse-error-end-tag-without-matching-open-element">end-tag-without-matching-open-element</span>
+    <span>parse error</span>. Ignore the token.</p>
    </dd>
 
    <dt>Anything else</dt>
    <dd>
 
-    <!-- can't get here with an EOF and a fragment case -->
+    <!-- can't get here with a fragment case -->
```

Comment:
Can you explain why this comment was wrong?

Reply:
Because we definitely can, consider: `<!DOCTYPE html><head><noscript>`

Reply:
I guess there was separate handling of EOF before, then it was refactored away, but comment left untouched.